crengine/src/lvstsheet.cpp

```diff
@@ -4499,6 +4503,35 @@ bool LVStyleSheet::parse( const char * str, bool higher_importance, lString32 co
     return _selectors.length() > 0;
 }
 
+bool LVStyleSheet::parseCharsetRule( const char * &str )
+{
+    // Parse rule '@charset' according https://developer.mozilla.org/en-US/docs/Web/CSS/@charset
+    if (!str || !*str)
+        return false;
+    skip_spaces( str );
+    if ( *str == '@' ) {
+        str++;
+        char word[64];
+        if ( parse_ident( str, word ) ) {
+            lString32 keyword(word);
```

Comment:
(Really minor remark: keyword and _charset could just be lString8 - as we parse `char *` and we don't need any of these to be anything else.)
Looks fine to me.

Reply:
> Really minor remark: keyword and _charset could just be lString8 - as we parse char * and we don't need any of these to be anything else.

Ok, but lString8 don't have lowercase() method, but I don't know if it is possible to use capital letters in the 'charset' keyword.

Reply:
> but lString8 don't have lowercase() method

Oh, ok, I thought it had it - but indeed, no lowercase() :/. Well, then, may be use lString32 for the check, better be safe, everything is possible in the wild :)
(As for the charset itself, it's supposed to be ASCII, so I guess it's ok to be lString8 and we'll get raw chars in it.)

Reply:
Fixed, ignoring case :)